FluentTerminal.App/Views/SettingsPages/About.xaml

```diff
@@ -12,7 +12,7 @@
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <Grid Grid.Row="1" Margin="24,0,0,24">
+        <Grid Grid.Row="1" Margin="24,0,0,0">
```

Comment:
Not as important, but we might consider defining a `<Thickness>` for this margin as well